cli.js

```diff
@@ -42,7 +48,34 @@ function init(data) {
 }
 
 if (process.stdin.isTTY || cli.flags.stdin === false) {
-	if (cli.flags.template) {
+	if (cli.flags.demo) {
+		const demoStr = [
```

Comment:
The whole demo thing should be moved to a top-level function and just called here.